contracts/sidechain/L2PoolManagerProxy.sol

```diff
@@ -44,12 +67,17 @@ contract L2PoolManagerProxy is NonblockingLzApp {
         IPoolManagerLite(poolManager).setOperator(_operator);
     }
 
+    /* -------------------------------------------------------------------
+       Core functions
+    ------------------------------------------------------------------- */
+
     /**
      * @notice Adds new pool directly on L2.
      * @param _gauge The gauge address.
      */
-    function addPool(address _gauge) external onlyOwner returns (bool) {
-        return _addPool(_gauge);
+    function addPool(address _gauge) external onlyKeeper returns (bool) {
```

Comment:
We still a version of this pools that is onlyOwner and does not need to check `isValidGauge`  to for example being able to add this https://app.aura.finance/#/8453/pool/11 or similar